_targets.R

```diff
@@ -0,0 +1,20 @@
+library(targets)
+source("functions.R")
+tar_option_set(packages = c("tidyverse", "stringr", "sbtools", "whisker"))
+
+list(
+  tar_target(
+    model_RMSEs_csv,
+    download_data('5d925066e4b0c4f70d0d0599', fetch_dir = '1_fetch/out', file_name = 'models_RMSE.csv'),
```

Comment:
I don't think this data ID does anything in the function right now. It's declared here but the argument is not used in the function itself. See my suggestion in `functions.R`